tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -237,6 +241,43 @@ function get_tdx_ovmf_path_from_distro() {
 	esac
 }
 
+function adjust_qemu_cmdline() {
```

Comment:
Ok, so I'd just like to check if I'm understanding this correctly:
If we are using a different install prefix then we check a new qemu binary with the `installation-prefix` suffix and in that we are calling the original binary with the -L parameter to set the path to our dest_dir version?

Reply:
Exactly!